lib/eslint.js

```diff
@@ -413,6 +413,7 @@ function prepareConfig(config) {
         globals: util.mergeConfigs({}, config.globals),
         env: util.mergeConfigs({}, config.env || {}),
         settings: util.mergeConfigs({}, config.settings || {}),
+        disableInlineConfig: config.disableInlineConfig || false,
```

Comment:
Actually, now I'm a bit confused. Sorry but I'm finding it a bit hard to figure out how to add an option for `CLIEngine` but not as a config option.

Where should I add this option without allowing `.eslintrc` files from adding it?